app/helpers/home_helper.rb

```diff
@@ -1,2 +0,0 @@
-module HomeHelper
-end
```

Comment:
This file was added in the first commit and then removed here. I suggest editing the first commit instead of creating a new one.

Reply:
Let me see how to delete this commit and amend it to the first one, its sounds tricky but I'll do it

Reply:
Let me know if you need help with this.